RxExample/RxExample/Examples/WikipediaImageSearch/Views/WikipediaSearchViewController.swift

```diff
@@ -58,36 +66,12 @@ class WikipediaSearchViewController: ViewController {
             .map { results in
                 results.map(SearchResultViewModel.init)
             }
-            .drive(resultsTableView.rx_itemsWithCellIdentifier("WikipediaSearchCell", cellType: WikipediaSearchCell.self)) { (_, viewModel, cell) in
+            .drive(resultController.tableView!.rx_itemsWithCellIdentifier("WikipediaSearchCell", cellType: WikipediaSearchCell.self)) { (_, viewModel, cell) in
                 cell.viewModel = viewModel
             }
             .addDisposableTo(disposeBag)
     }
 
-    func configureKeyboardDismissesOnScroll() {
```

Comment:
Is there some reason why we deleted `configureKeyboardDismissesOnScroll` and `configureNavigateOnRowClick `?